src/model/guild/member.rs

```diff
@@ -564,3 +566,30 @@ pub struct PartialMember {
     #[cfg_attr(docsrs, doc(cfg(feature = "unstable_discord_api")))]
     pub permissions: Option<String>,
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[non_exhaustive]
+pub struct ThreadMember {
+    /// The id of the thread.
+    pub id: ChannelId,
+    /// The id of the user.
+    pub user_id: Option<UserId>,
+    /// The time the current user last joined the thread.
+    pub join_timestamp: DateTime<Utc>,
+    /// Any user-thread settings, currently only used for notifications
+    pub flags: ThreadMemberFlags,
+}
+
+/// Describes extra features of the message.
+#[derive(Copy, PartialEq, Eq, Clone, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct ThreadMemberFlags {
+    pub bits: u64,
+}
+
+#[cfg(feature = "model")]
```

Comment:
The `model` feature should only be applied on methods belonging to model types.
```suggestion

```

Reply:
Oh well it is also restricted to model feature on MessageFlags, it is an error I guess?

Reply:
Yes, it's a wrong use-case for the feature.